lib/1.4/dml-builtins.dml

```diff
@@ -893,10 +893,10 @@ configuration [`"required"` | `"optional"` | `"pseudo"` | `"none"`]
 
 Specifies how Simics treats the automatically created
 [attribute](#attribute-objects)
-corresponding to the connect object. Note that for array objects,
-setting this parameter to `"optional"` has the effect that each
-instance of the connect can be individually set to a nil value.
-</dd><dt>
+corresponding to the connect object. The default value is `"optional"`.
+
+The attribute can be set to a nil value only if this parameter is `"optional"`
+or `"required"`. In an array of connects, this applies element-wise.</dd><dt>
```

Comment:
... _or_ required? Unless I'm being very dumb right now, that's not true.

Reply:
Thanks! I meant `"pseudo"`.